### PR TITLE
Added Recobundles + changed ants instructions

### PR DESCRIPTION
**Pull request by EmmaRenauld**

### CODE REVIEW

File: source/our_tools/recobundles.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    tractoflow_folder=my_dir
+    subject_list=my_subject_list.txt
+    model_T1=my_model.nii.gz  # Ex: hcp_models/mni_masked.nii.gz
+
+    # Filtering options. Change as needed.
+    minL=20
+    maxL=200
+
+    # Recobundles options. Change as needed.
+    nb_total_tests=9    # Nb bundles * nb_per_bundle (see json) = max total tests
```
Comment (frheault) [inline source/our_tools/recobundles.rst]: Instead of tests, I would put executions

File: source/our_tools/recobundles.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    seed=0              # Random generator.
+    minimal_vote=0.5    # Saving streamlines if recognized often enough
+
+
+    while IFS= read -r subj; do
+        echo "Running subject $subj"
+
+        # Defining subj folders
+        subj_folder=$tractoflow_folder/$subj
+        rbx_folder=$subj_folder/Recobundles
```
Comment (frheault) [inline source/our_tools/recobundles.rst]: RecobundlesX ?

**Comment by EmmaRenauld** [comment]
> Did you setup ANTs like in the instructions on your computer?
> Does it work with the *.sh scripts (e.g antsRegistrationSyNQuick.sh)

Yes. ANTs computes fine but as I told you, the bashrc part doesn't work... But's it copy-paste from their website! I might need help to understand this. Tell me when you have some time.

**Comment by EmmaRenauld** [comment]
@frheault 

I installed ANTs (yé!), I ran Recobundles (yé!) on my HCP data preprocessed with Tractoflow (yé!), I checked the mosaic to see if I liked the results (yé!).

So eveything looks good to me for final review.  To help you, I am sending you a printscreen of the html version.
https://drive.google.com/file/d/13KulEWcrEZL5uTN2JAGnmM6RqgerssC3/view?usp=sharing
https://drive.google.com/file/d/1b_tVV8oAquVtFMS-wKLQlYWBPfpFgZun/view?usp=sharing

I think the Recobundles page should have more explanation on what the parameters are and how they work. Could you prepare me some sentences that I could add?

@mdesco  Anything else to add?

File: source/our_tools/recobundles.rst [hunk 3]
```diff
@@ -0,0 +1,95 @@
+RecobundlesX
+============
+
+Here is a typical example of a script to run our scilpy's version of Recobundles on a batch of subjects. Dipy also has its version, and you can find the explanation `here <https://dipy.org/documentation/0.16.0./examples_built/bundle_extraction/>`_.
```
Comment (mdesco) [inline source/our_tools/recobundles.rst:4]: Can we add a line here to say that dipy's version is single atlas, single parameter.
Our version, is multi-atlas and multi-parameter, and as shown in Rheault PhD 2020, is more robust.

Moreover, the user does not need to make unecessary copy of the whole-brain tractogram while being registered.

@frheault help please

Comment (EmmaRenauld) [reply source/our_tools/recobundles.rst]: @mdesco , modification is ok for you?

**Pull request merged**